### DISCUSSION TITLE
user_join 이벤트를 사용하여 유저가 옵챗에 입장 시 봇이 메세지를 띄우게 하고 싶습니다.

### DISCUSSION THREAD

**Discussion by djkirop342** [storycraft/node-kakao#806] [Q&A]
<img width="803" alt="스크린샷 2021-11-18 오후 1 05 12" src="https://user-images.githubusercontent.com/47454392/142349898-0b0441ac-f411-41ad-a24e-8e3692b2bbe1.png">

user_join 이벤트를 사용하여 옵챗에 유저가 입장 시
이렇게 처리를 하려고 하는데 channel.sendText 함수가 없다고 에러가 나는데
혹시 다른 함수가 있는건지요..??

profile_changed 이벤트에서는 저 함수가 먹혔는데
user_join 이벤트에서는 먹히지 않네요...

찾아보니 channel_join 도 있던데
user_join 과 channel_join 차이가 뭔지도 궁금합니다.

**Comment by tadashi74** [comment]
만약 사용하시는 버전이 v4면 `channel.sendChat('어서오세요')` 로 하셔야 합니다.

**Reply by djkirop342** [reply]
지금은 v3 버전을 쓰고 있는데 혹시 v3에서 저 방식은 사용 할 수 없는 방법일까요?
